command-line-tools/getLatestGitRepository.sh

```diff
@@ -2,6 +2,15 @@
 # use sh getLatestGitRepository.sh ; to download repository from latest snapshots
 # use sh getLatestGitRepository.sh githubBranch; to download repository from latest snapshots and to switch to a specific branch/tag
 
+while getopts ":s:r:w" opt; do
+    case $opt in
+        s)
+            URL_SNAPSHOTS="$OPTARG"
+            ;;
```

Comment:
handling other cases would be nice

    *)
        echo "Usage: ..."
        ;;